backend/lambda-functions/deleteSessions/handlers/handlers.go

```diff
@@ -217,6 +217,11 @@ func (h *handlers) DeleteSessions(ctx context.Context, projectId int, startDate
 		return
 	}
 
+	if err := h.storageClient.CleanupRawEvents(ctx, projectId); err != nil {
+		log.WithContext(ctx).Error(err)
```

Comment:
should we have some more resilient error handling here so that we still delete sessions if this fails?
![Screenshot from 2024-09-26 09-44-02](https://github.com/user-attachments/assets/28ae3593-09d6-4986-a783-fa734e1be6e4)

Reply:
👍 updated to catch not exists errors, removed the `return` and am just logging errors if something happens while cleaning up `raw-events`